http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientUriBuilder.java

```diff
@@ -0,0 +1,597 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.wildfly.security.http.oidc;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+/**
+ * @author <a href="mailto:bill@burkecentral.com">Bill Burke</a>
+ * @version $Revision: 1 $
+ */
+public class OidcClientUriBuilder {
```

Comment:
This feels a bit more justified to be public but if it is only in this package maybe it could be reduced.

Reply:
fixed, thanks!